src/CreeDictionary/CreeDictionary/views.py

```diff
@@ -54,20 +55,38 @@ def entry_details(request, lemma_text: str):
         - paradigm-size (default is BASIC) to specify the size of the paradigm
 
     :param request: accepts query params `pos` `inflectional_category` `analysis` `id` to further specify query_string
-    :param lemma_text: the exact form of the lemma (no spell relaxation)
+    :param slug: the stable unique ID of the lemma
     """
 
-    lemma = Wordform.objects.filter(text=lemma_text, is_lemma=True)
-    if additional_filters := disambiguating_filter_from_query_params(request.GET):
```

Comment:
Ah, this view is much clearner in general!

Please delete the documentation in the docstring related to paradigm sizes 👆🏼 👆🏼 👆🏼